airbyte-integrations/connectors/source-google-ads/source_google_ads/spec.json

```diff
@@ -67,6 +67,31 @@
         "maximum": 1095,
         "default": 14,
         "examples": [14]
+      },
+      "custom_query": {
```

Comment:
```suggestion
      "custom_queries": {
```

Reply:
this will require also to update string literals in source.py where we use it - I will update this after git creds will be upgraded